.fossa.yml

```diff
@@ -0,0 +1,38 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.com to learn more
+
+version: 2
+cli:
+  server: https://app.fossa.com
+  fetcher: custom
+  project: management-api-for-apache-cassandra
+analyze:
+  modules:
+  - name: DataStax Management API for Apache Cassandra
+    type: mvn
+    target: pom.xml
+    path: .
+  - name: datastax-mgmtapi-agent-3.x
+    type: mvn
+    target: pom.xml
+    path: management-api-agent-3.x
+  - name: datastax-mgmtapi-agent-4.x
+    type: mvn
+    target: pom.xml
+    path: management-api-agent-4.x
+  - name: datastax-mgmtapi-agent-common
+    type: mvn
+    target: pom.xml
+    path: management-api-agent-common
+  - name: datastax-mgmtapi-agent-dse-6.8
+    type: mvn
+    target: pom.xml
+    path: management-api-agent-dse-6.8
```

Comment:
For the DSE agent, the project has to be built with the Maven profile `dse` enabled. I don't know i that has to be accounted for in the FOSSA config here or not. From the docs, it looks like it's going to run `mvn dependency:tree` to analyze things. If you don't do `mvn dependency:tree -Pdse`, it won't include the DSE agent. I'll look at the report to see if it actually is pickup up the DSE agent.

Reply:
Gotcha, I'd have to do some digging or talk to Fossa to see if we can change the way it behaves for a certain module like that.  It certainly won't stop Fossa from scanning, so I think as it is it's OK, but we can see if we can make it a bit more accurate if necessary.